File: app/src/main/java/cn/surine/schedulex/app_widget/normal_week/PaintConfig.java
package cn.surine.schedulex.app_widget.normal_week;

import android.graphics.Color;

import cn.surine.schedulex.data.helper.DataHandler;

/**
 * Intro：
 * 周视图绘制配置项
 *
 * @author sunliwei
 * @date 2020/6/8 17:32
 */
public class PaintConfig {
    public int slotHeight = DataHandler.abt.getInstance().getCurSchedule().itemHeight;
    public int itemRadius = 3;
    public int gapHeight = 3;
    public int gapWidth = 3;
    public int courseTextColor = Color.WHITE;
    public float itemStroke = 1.2F;
}

File: app/src/main/java/cn/surine/schedulex/app_widget/normal_week/PaintConfig.kt
package cn.surine.schedulex.app_widget.normal_week

import android.graphics.Color
import cn.surine.schedulex.data.helper.DataHandler

/**
 * Intro：
 * 周视图绘制配置项
 *
 * @author sunliwei
 * @date 2020/6/8 17:32
 */
class PaintConfig {
    var slotHeight: Int = DataHandler.getCurSchedule()?.itemHeight ?: 80
    var itemRadius = 3
    var gapHeight = 3
    var gapWidth = 3
    var courseTextColor = Color.WHITE
    var itemStroke = 1.2f
    var isShowWeek: Boolean = DataHandler.getCurSchedule()?.isShowWeekend ?: false
}